src/GameObject.cpp

```diff
@@ -79,29 +79,11 @@ void GameObject::setYAxisRotation(float angle) {
 void GameObject::calculateAndSetInitialRotation() {

```

Comment:
Okay @joncatanio, I moved the code to the BunnyPhysicsComponent, let me know what you think.  Still not entirely happy with the PhysicsInit stuff since it should probably be more standardized but I'll adhere to the [YAGNI principle](https://en.wikipedia.org/wiki/You_aren't_gonna_need_it) for now.